api/server.py

```diff
@@ -186,6 +224,5 @@ async def exception_handler(request: Request, e: Exception):
         'api.server:app',
         host='0.0.0.0',
         port=int(os.getenv('PORT', '8000')),
-        # debug=True,
-        reload=True,
+        reload=SM_ENVIRONMENT == 'local',
```

Comment:
My understanding is this doesn't run in production, uvicorn selects the right app to deploy with:
```
uvicorn api.server:app
```

Reply:
Ah yeah, of course! Thanks for pointing out - I've reverted this bit of the change